debian/changelog

```diff
@@ -1,3 +1,54 @@
+ubuntu-advantage-tools (27.9~22.04.1) jammy; urgency=medium
+
+  * d/rules
+    - remove trusty specific code
+    - remove ua-license-check.{timer,service,path}
+    - install ubuntu-advantage.service
+    - only on xenial: install ubuntu-advantage-cloud-id-shim.service
+  * d/tools.postinst
+    - remove trusty specific code
+    - print warnings if /etc/os-release doesn't have required fields
+    - hardcode service list instead of exec-ing python3 for old migration
+    - refactor python to avoid instantiating UAConfig extra times
+    - refactor python to always use messages module for strings
+    - rm the old marker file that triggered ua-license-check.path
+    - remove unnecessary deb-systemd-helper check in ua-messaging cleanup
+    - clean up old ua-license-check state
+    - run new cloud-id-shim script
+  * d/tools/postrm
+    - clean up ubuntu-advantage-daemon log files
+  * New upstream release 27.9 (LP: #1973099)
+    - cli:
```

Comment:
I think we can advertise that we have a new command that updates MOTD and APT messages. Even though they are already on the refresh section